resultsdbupdater/utils.py

```diff
@@ -19,6 +19,16 @@
     format=log_format, level=CONFIG.get('resultsdb-updater.log_level'))
 
 
+def update_publisher_id(data, msg):
+    """
+    Sets data['publisher_id'] to message publisher ID (JMSXUserID) if it
+    exists.
+    """
+    msg_user_id = msg['headers'].get('JMSXUserID')
```

Comment:
How about changing `msg_user_id` to `msg_publisher_id` or `jmsx_user_id`?

Reply:
Can we please add a log statement if the `JMSXUserID` is missing?

Reply:
> How about changing `msg_user_id` to `msg_publisher_id` or `jmsx_user_id`?

From my perspective anything with `msg` or even `jmsx` would be more confusing in test result data (added comment also to the related JIRA).

> Can we please add a log statement if the `JMSXUserID` is missing?

I'm planning to do that as part of a different task. Also, it does seem that it's missing in messages in Fedora infrastructure.

Reply:
> > How about changing `msg_user_id` to `msg_publisher_id` or `jmsx_user_id`?
> 
> From my perspective anything with `msg` or even `jmsx` would be more confusing in test result data (added comment also to the related JIRA).

Oh, I misunderstood. If it's just a variable name change, I can rename it to `msg_publisher_id`.